src/bugreporter/odemis_bugreporter.py

```diff
@@ -66,6 +71,16 @@
 BACKEND_STOPPED = 2
 BACKEND_STARTING = 3
 
+# Topic IDs are used to indicate which "department" will be taking care of the ticket.
+# They can be found from the urls at https://support.delmic.com/scp/helptopics.php
```

Comment:
The link doesn't bring me to the helptopics, it just redirects to https://support.delmic.com/scp/index.php. And does it make sense to add a fastem topic?